artemis/templates/configmaps/artemis-usermanagement-configmap.yml

```diff
@@ -10,5 +10,11 @@ data:
   {{- else }}
   artemis.user-management.use-external: "false"
   {{- end }}
-
-
+  #TODO: is the if block okay like this? based on ansible template or already if just url for instance is set?
```

Comment:
Not 100 percent sure if the logic makes sense. Used the ansible template as a base.